jib-cli/src/main/java/com/google/cloud/tools/jib/cli/JibCli.java

```diff
@@ -67,6 +74,24 @@ static void logTerminatingException(
             + "\u001B[0m");
   }
 
+  /**
+   * Writes image details (imageId, digest, tags, etc.) to a json file (jib-image.json), if the
+   * destination of the json is provided.
+   *
+   * @param imageJsonOutputPath optional path to jib-image.json (for example,
+   *     path/to/json/jib-image.json)
+   * @param jibContainer the {@link JibContainer} to derive image details from
+   * @throws IOException if error occurs when writing to the json file.
+   */
+  static void writeImageJson(Optional<Path> imageJsonOutputPath, JibContainer jibContainer)
+      throws IOException {
+    if (imageJsonOutputPath.isPresent()) {
+      ImageMetadataOutput metadataOutput = ImageMetadataOutput.fromJibContainer(jibContainer);
+      Files.write(
+          imageJsonOutputPath.get(), metadataOutput.toJson().getBytes(StandardCharsets.UTF_8));
```

Comment:
It doesn't fail if the file already exists?

Reply:
The file gets overwritten if it already exists